internal/multigitter/run.go

```diff
@@ -299,8 +300,10 @@ func (r *Runner) interactive(ctx context.Context, dir string, repo domain.Reposi
 				return err
 			}
 		case 'r':
+			fmt.Println("Rejected, continuing...")
 			return errRejected
 		case 'a':
+			fmt.Println("Accepted, creating pull request...")
```

Comment:
"creating pull request" is not necessarily true since using the `--skip-pr` would result in a commit directly to the main branch.

Reply:
@lindell thanks - reworded this to "proceeding...".